### PR TITLE
Update ScheduleList view

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
screen shot 2016-09-23 at 2 02 22 pm

**Comment by justinvdm** [comment]
@peterp ready for review

File: index.android.js [hunk 1]
```diff
@@ -1,5 +1,17 @@
 import React from 'react';
-import MentorshipApp from 'src/app';
+import ScheduleList from 'src/views/ScheduleList';
 import { AppRegistry } from 'react-native';
 
-AppRegistry.registerComponent('app', () => () => <MentorshipApp />);
+AppRegistry.registerComponent('app', () => () => (
```
Comment (peterp) [inline index.android.js]: Might wanna take this out 💨

Comment (justinvdm) [inline index.android.js]: /o\ knew this would happen at some point, sorry, will remove

**Pull request merged**